slurm.sh

```diff
@@ -166,8 +181,8 @@ slurm_completed_job_status_from_sacct() {
     
     # Wait for log file to be complete
 
-    local logComplete=1
-    checkCount=0
+    # local logComplete=1
+    # checkCount=0
```

Comment:
`logComplete` is not used in the script, but `checkCount` is defined and used few lines above. Please make sure deleting it here does not break anything

Reply:
The local variable `checkCount` used in previous lines is different, and is then reset for the succeeding lines that check log completion. It is not used again for the rest of the function.